doc/polyglossia.tex

```diff
@@ -44,14 +43,77 @@
 \def\Ie{\textit{I.e.,}\xspace}
 \def\etc{\@ifnextchar.{\textit{etc}}{\textit{etc.}\@\xspace}}
 
+%% Commands for documenting options
+\ExplSyntaxOn
+
+\NewDocumentCommand \pgoption {m}
+  {
+    \textcolor {pgblue} { \bfseries #1 }
+  }
+
+\NewDocumentCommand \pgdefault {m}
+  {
+    \textit{#1}
+  }
+
+\NewDocumentCommand \pgdefaultchoice {m}
+  {
+    \underline{#1}
```

Comment:
No underlining, please. I think a preceding asterisk is quite common to mark defaults.

Reply:
I saw "italic for default values, regular for the others" in other packages and liked it. But I'm not sure if a preceding asterisk is the commonish and therefore preferable style in LaTeX packages.

Reply:
Note that we have italic for default values of given keys. We need something else for preset options, and the asterisk seems a good choice

Reply:
Oh, sorry for the noise. I second the asterisk in this case.

Reply:
> Note that we have italic for default values of given keys. We need something else for preset options, and the asterisk seems a good choice

I think it is the other way around: We use italic type for preset values of options (as in the current manual) and we need something else for values of keys that are called without a value.

Is it enough to replace `\underline{#1}` by `*#1` in this command or do you prefer something more sophisticated?

Reply:
This seems sufficient.

Reply:
OK, this one is easy.